ReSharper.FSharp/test/src/FSharp.Tests.Common/src/ZoneMarker.fs

```diff
@@ -17,3 +18,7 @@ type PsiFeatureTestZoneActivator() =
 [<ZoneActivator>]
 type FSharpZoneActivator() =
     interface IActivate<ILanguageFSharpZone>
+
+[<ZoneActivator>]
+type SyntaxHighlightingZoneActivator() =
+    interface IActivate<ISyntaxHighlightingZone>
```

Comment:
F# now allows multiple implementations of the same interface (it didn't allow it when `PsiFeatureTestZoneActivator` and `FSharpZoneActivator` were implemented), maybe it all could be inside a single type?